index.html

```diff
@@ -24,18 +24,23 @@ <h1>Skinclusive<br>inclusive of your skin.</h1>
 			
 			<!-- This is the individual text box-->
 			<div class="subtitle">
-				<p>Where to start?</p>
+				<p>Browse our top articles</p>

```

Comment:
In this version, I removed the left and right boxes and instead added the same flexbox seen in version 1's "article" HTML code.